ui/src/timeMachine/actions/queries.ts

```diff
@@ -147,15 +148,15 @@ export const executeQueries = (abortController?: AbortController) => async (
     // https://github.com/influxdata/idpe/issues/6240
 
     const startTime = window.performance.now()
+    const startDate = Date.now()
 
     pendingResults.forEach(({cancel}) => cancel())
-    reportSimpleQueryPerformanceEvent('executeQueries queries start')
 
     pendingResults = queries.map(({text}) => {
       reportQueryPerformanceEvent({
-        timestamp: Date.now() * 1000000,
-        fields: {},
-        tags: {event: 'executeQueries queries', query: text},
+        timestamp: toNano(Date.now()),
```

Comment:
like we mentioned IRL, move this inside of `runQuery` so we have all `runQuery` calls being tracked, and then call to `executeQueries querying` will handle this part :+1:

Reply:
this turns out way harder than I had expected, so skipping for now